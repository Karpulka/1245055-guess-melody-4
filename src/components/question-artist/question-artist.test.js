import React from "react";
import renderer from "react-test-renderer";
import QuestionArtist from "./question-artist";

it(`Render QuestionArtist`, () => {
  const tree = renderer
    .create(<QuestionArtist/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
