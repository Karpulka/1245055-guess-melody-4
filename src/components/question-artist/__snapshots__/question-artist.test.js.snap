// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render QuestionArtist 1`] = `
<section
  className="game game--artist"
>
  <header
    className="game__header"
  >
    <a
      className="game__back"
      href="#"
    >
      <span
        className="visually-hidden"
      >
        Сыграть ещё раз
      </span>
      <img
        alt="Угадай мелодию"
        className="game__logo"
        src="img/melody-logo-ginger.png"
      />
    </a>
    <svg
      className="timer"
      viewBox="0 0 780 780"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        className="timer__line"
        cx="390"
        cy="390"
        r="370"
        style={
          Object {
            "filter": "url(#blur)",
            "transform": "rotate(-90deg) scaleY(-1)",
            "transformOrigin": "center",
          }
        }
      />
    </svg>
    <div
      className="game__mistakes"
    >
      <div
        className="wrong"
      />
      <div
        className="wrong"
      />
      <div
        className="wrong"
      />
    </div>
  </header>
  <section
    className="game__screen"
  >
    <h2
      className="game__title"
    >
      Кто исполняет эту песню?
    </h2>
    <div
      className="game__track"
    >
      <div
        className="track"
      >
        <button
          className="track__button track__button--play"
          type="button"
        />
        <div
          className="track__status"
        >
          <audio
            src=""
          />
        </div>
      </div>
    </div>
    <form
      className="game__artist"
    >
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="answer"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="one"
        />
        <label
          className="artist__name"
          htmlFor="answer"
        >
          <img
            alt="one"
            className="artist__picture"
            src="pic-one"
          />
          one
        </label>
      </div>
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="answer"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="two"
        />
        <label
          className="artist__name"
          htmlFor="answer"
        >
          <img
            alt="two"
            className="artist__picture"
            src="pic-two"
          />
          two
        </label>
      </div>
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="answer"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="three"
        />
        <label
          className="artist__name"
          htmlFor="answer"
        >
          <img
            alt="three"
            className="artist__picture"
            src="pic-three"
          />
          three
        </label>
      </div>
    </form>
  </section>
</section>
`;
